appium/webdriver/extensions/location.py

```diff
@@ -51,6 +52,7 @@ def set_location(
             longitude: String or numeric value between -180.0 and 180.0
             altitude: String or numeric value (Android real device only)
             speed: String or numeric value larger than 0.0 (Android real devices only)
+            satellites: String or numeric value since Appium 1.22.0
```

Comment:
We could have a more precise documentation string here:
Count of active GPS satellites in range 1..12. Only works for Emulators.